src/main/java/practice/StreamPractice.java

```diff
@@ -75,6 +106,11 @@ public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
      * parametrized with Candidate in CandidateValidator.
      */
     public List<String> validateCandidates(List<Candidate> candidates) {
-        return Collections.emptyList();
+        CandidateValidator candidateValidator = new CandidateValidator();
```

Comment:
```suggestion
        CandidateValidator candidateValidator = new CandidateValidator();
```
add this as private field before all methods